<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="dbworkplaceview" name="workplace">
    <property name="geometry">
     <rect>
      <x>213</x>
      <y>-3</y>
      <width>641</width>
      <height>561</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777195</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>211</width>
      <height>561</height>
     </rect>
    </property>
    <widget class="QToolButton" name="drawlineButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>line</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icon/直线.png</normaloff>:/icon/直线.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="clearButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>47</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QToolButton" name="selectButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>47</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>单选</string>
     </property>
    </widget>
    <widget class="QToolButton" name="rectbutton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>rect</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icon/矩形.png</normaloff>:/icon/矩形.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="fillButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>fill</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icon/刷子.png</normaloff>:/icon/刷子.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="drawpolygon">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>poly</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icon/polygon.png</normaloff>:/icon/polygon.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="drawellipse">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>elli</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icon/椭圆.png</normaloff>:/icon/椭圆.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="paintersetting" name="setting" native="true">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>419</y>
       <width>181</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="deletebutton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>dele</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icon/删除操作1.png</normaloff>:/icon/删除操作1.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="multiselect">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>47</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>多选</string>
     </property>
    </widget>
    <widget class="QToolButton" name="drawbezier">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Bezier</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="bezierVnum">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>290</y>
       <width>46</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>4</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
    </widget>
    <widget class="QToolButton" name="drawBspline">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>B-spline</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="bsplinenum">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>350</y>
       <width>46</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>6</number>
     </property>
     <property name="maximum">
      <number>8</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="bsplinek">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>370</y>
       <width>46</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>3</number>
     </property>
     <property name="value">
      <number>3</number>
     </property>
    </widget>
    <widget class="QToolButton" name="drawtriangle">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>180</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icon/三角形.png</normaloff>:/icon/三角形.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="drawcircle">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>180</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>circle</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icon/圆1.png</normaloff>:/icon/圆1.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="bezierFit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>240</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>曲线拟合</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="bezierFitVnum">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>290</y>
       <width>46</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>4</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>855</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>dbworkplaceview</class>
   <extends>QGraphicsView</extends>
   <header>dbworkplaceview.h</header>
  </customwidget>
  <customwidget>
   <class>paintersetting</class>
   <extends>QWidget</extends>
   <header location="global">paintersetting.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
